### PR TITLE
Bump hardhat to 2.22

### PR DESCRIPTION
**Pull request by kronosapiens**
Closes https://github.com/JoinColony/colonyNetwork/issues/1228

### CODE REVIEW

File: scripts/setup-bridging-contracts.js [hunk 1]
```diff
@@ -145,8 +146,8 @@ async function setupBridging(homeRpcUrl, foreignRpcUrl) {
   return { gnosisSafe, bridgeMonitor, zodiacBridge, homeBridge, foreignBridge, homeColonyBridge, foreignColonyBridge };
 }
 
-async function getSig(provider, account, dataHash) {
-  const sig = await provider.send("eth_sign", [account, dataHash]);
```
Comment (area) [inline scripts/setup-bridging-contracts.js:149]: This has been needed to due to an issue being tracked here: https://github.com/NomicFoundation/edr/issues/455

While signing messages in the way you've done here solves the issue for our codebase, `eth_sign` is still not callable on a Hardhat node, which is required for the CDApp development environment. As a result, this PR will need to wait until Hardhat has fixed the problem, and which point I would revert the changes related to this to catch similar issues in the future (which the CDApp can't work around).

**Review by area — COMMENTED** [review]
Sadly, can't approve for reasons beyond your control.

**Comment by area** [comment]
Just noting that the `eth_sign` issue has now been fixed on the EDR side, so once the next version is released, we should be able to move to the latest hardhat.

**Comment by kronosapiens** [comment]
Closing in favor of #1254

**Pull request closed without merging**